agent-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "totalValueAndLiquidity": {
```

Comment:
The other keys in agent-config.json are written as "total-value-and-liquidity"

Reply:
Perhaps `total_value_and_liquidity` to match the format that @arbitrary-cr0 is using in his branch.